[docs-only] update ociswrapper document (#11227)
owncloud/ocis: tests/ociswrapper/README.md

## oCIS Wrapper

A tool that wraps the oCIS binary and provides a way to re-configure the running oCIS instance.

> It is essentially a wrapper layer to dynamically configure and extend a running oCIS instance by sending a JSON payload with updated environment variables.

Reviewer: ```suggestion
It is essentially a wrapper layer to dynamically configure and extend a running oCIS instance by sending environment variables as a JSON payload.
```

> A tool that wraps the oCIS binary and provides a way to re-configure the running oCIS instance.
> It is essentially a wrapper layer to dynamically configure and extend a running oCIS instance by sending a JSON payload with updated environment variables.

Reviewer 2: ```suggestion
The oCIS wrapper is a tool that wraps the oCIS binary and allows to dynamically reconfigure or extend the running instance. This is done by sending JSON payloads with updated environment variables.
```
Note that this supersedes the former suggestable commit from @saw-jan

> It allows live configuration updates without needing to restart the oCIS server manually.

Reviewer: ```suggestion
```

When run, **ociswrapper** starts an API server that exposes some endpoints to re-configure the oCIS server.

### Usage

1.  Build the wrapper
# working dir: ocis repo root dir

    ```bash
    cd tests/ociswrapper

>     make build

Reviewer: these can be left unchaged

    ```

    ```bash
     --bin string              Full oCIS binary path (default "/usr/bin/ocis")
     --url string              oCIS server url (default "https://localhost:9200")
     --retry string            Number of retries to start oCIS server (default "5")
     -p, --port string         Wrapper API server port (default "5200")
     --admin-username string   admin username for oCIS server
     --admin-password string   admin password for oCIS server
     --skip-ocis-run           Skip running oCIS server
    ```

> Access the API server at `http://localhost:5200`.

Reviewer: Could you update the line:47 below `**ociswrapper** exposes two endpoints:` to `**ociswrapper** exposes the following endpoints:`?
